### PR TITLE
Unify Taskwarrior hook tutorial into separate page & extend it

### PR DESCRIPTION
**Pull request by xerus2000**
The documentation on the hook was a bit all over the place, and not very detailed. I thus improved and extended to the best of my knowledge.

### CODE REVIEW

**Comment by xerus2000** [comment]
@lauft can we merge?

File: html/docs/taskwarrior.html [hunk 1]
```diff
@@ -53,39 +53,56 @@
...
-              Whenever you <code>start</code> a task in Taskwarrior, the hook
-              script will start time tracking in Timewarrior. 
+              Below you can see an example usage: create a task, start it,
+              complete it and automatically track it.
+            </p>
 
-              When the task is <code>stop</code>ped, or completed, the hook
-              script will stop time tracking  in Timewarrior.
+            <p>
+                <img src="/docs/images/taskwarrior1.png" class="img-responsive" />
```
Comment (lauft) [inline html/docs/taskwarrior.html:105]: I think a better name for this picture would be `taskwarrior-on-modify-hook-example.png`.

**Comment by xerus2000** [comment]
@lauft fixed the review comments. I adjusted the documentation to match the feature set in https://github.com/GothenburgBitFactory/timewarrior/pull/329.

**Comment by lauft** [comment]
Looks good. It will be merged as soon as GothenburgBitFactory/timewarrior#329 is closed.

**Comment by lauft** [comment]
I have adapted your modifications to the new Hugo layout and merged them. Thanks for your contribution. 👍🏻

**Pull request closed without merging**